lib/plugins/aws/package/compile/events/websockets/lib/deployment.js

```diff
@@ -24,6 +24,12 @@ module.exports = {
         },
       },
     });
+    const { apiGateway } = this.serverless.service.provider;
+    if (apiGateway && apiGateway.websocketApiId) {
+      this.serverless.service.provider.compiledCloudFormationTemplate.Resources[
```

Comment:
For readability it'll be nice if we assign `this.serverless.service.provider.compiledCloudFormationTemplate.Resources` first to some local var, and then rely on it at L16 and L29

Reply:
done